package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "amplify-ios",
+  "version": "0.12.0",
+  "description": "Amplify package",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "release": "standard-version"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aws-amplify/amplify-ios.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/aws-amplify/amplify-ios/issues"
+  },
+  "homepage": "https://github.com/aws-amplify/amplify-ios#readme",
+  "devDependencies": {
+    "standard-version": "^8.0.0"
+  }
+}
```

Comment:
i wonder what this is used for? i'm assuming this change is coming from a merge from some other commit

Reply:
Removed